### DISCUSSION TITLE
How do i create response links?

### DISCUSSION THREAD

**Discussion by bryanjtc** [domaindrivendev/Swashbuckle.AspNetCore#3072] [Q&A] [answered]
I wanted to know how to define links for responses using Swashbuckle.
https://swagger.io/docs/specification/links/

**Comment by martincostello** [comment, marked as answer]
There's no automatic support for this built in. You'll need to write a [custom filter](https://github.com/domaindrivendev/Swashbuckle.AspNetCore?tab=readme-ov-file#extend-generator-with-operation-schema--document-filters) to add links to your OpenAPI responses: [OpenApiResponse.Links](https://github.com/microsoft/OpenAPI.NET/blob/11925be70ed772f8cbede6f4a9b54d1559851a49/src/Microsoft.OpenApi/Models/OpenApiResponse.cs#L32-L37).
